components/Icons/SmallArrowLeft.js

```diff
@@ -0,0 +1,21 @@
+// Import of styled component
+import { StyledButtonSVG } from "./Icons.styled";
+
+export function SmallArrowLeft() {
+  return (
+    <StyledButtonSVG
+      xmlns="http://www.w3.org/2000/svg"
+      fill="none"
+      viewBox="0 0 24 24"
+      strokeWidth={1.5}
+      stroke="currentColor"
+      className="w-6 h-6"
+    >
+      <path
+        strokeLinecap="round"
+        strokeLinejoin="round"
+        d="M10.5 19.5L3 12m0 0l7.5-7.5M3 12h18"
+      />
+    </StyledButtonSVG>
```

Comment:
You are handling SVG icons kind of unfavorable, see if this approach can help you to more order:

https://codesandbox.io/s/dreamy-joana-13ij6o?file=/src/App.js

Reply:
I will look into it next when I do a styling overhaul.
I know about variants but I don't want to concentrate my time on icons, yet.